setup.py

```diff
@@ -0,0 +1,39 @@
+from setuptools import setup, find_packages
+
+with open('README.rst') as readme_file:
+    readme = readme_file.read()
+
+requirements = [
+    'girder>=3.0.0a1'
+]
+
+setup(
+    author='Kitware Inc.',
+    author_email='roni.choudhury@kitware.com',
+    classifiers=[
+        'Development Status :: 2 - Pre-Alpha',
+        'License :: OSI Approved :: Apache Software License',
+        'Natural Language :: English',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8'
+    ],
+    description='A Girder plugin for Tulane HistomicsTK',
+    install_requires=requirements,
+    license='Apache Software License 2.0',
+    long_description=readme,
+    long_description_content_type='text/x-rst',
+    include_package_data=True,
+    keywords='girder-plugin, girder_tulane',
+    name='girder_tulane',
+    packages=find_packages(exclude=['test', 'test.*']),
+    url='https://github.com/arclamp/girder-tulane',
+    version='0.1.0',
```

Comment:
For scm versions, this gets replaced with
```
use_scm_version={'local_scheme': prerelease_local_scheme},
```
and then we add a function to this file:
```
def prerelease_local_scheme(version):
    """
    Return local scheme version unless building on master in CircleCI.

    This function returns the local scheme version number
    (e.g. 0.0.0.dev<N>+g<HASH>) unless building on CircleCI for a
    pre-release in which case it ignores the hash and produces a
    PEP440 compliant pre-release version number (e.g. 0.0.0.dev<N>).
    """
    from setuptools_scm.version import get_local_node_and_date

    if os.getenv('CIRCLE_BRANCH') in ('master', ):
        return ''
    else:
        return get_local_node_and_date(version)
```
assuming that we'll add circle-ci tests.  Otherwise you can just use `get_local_node_and_date(version)` directly.

Reply:
I have been using GitHub Actions on recent projects, mainly because it integrates better with GitHub repos (almost by definition). Let me know if you have a preference.

I will copy your function word-for-word for now and we can adjust if needed as we implement CI.

Reply:
See 612d1471779c9b491daa4bae24593f63ea1b840b

Reply:
> I have been using GitHub Actions on recent projects, mainly because it integrates better with GitHub repos (almost by definition). Let me know if you have a preference.

I haven't used GitHub Actions, but if we can run tox there, then I don't really care one way or the other.